src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -598,6 +597,10 @@ public void afterSave(@NonNull SCMNavigatorOwner owner) {
         }
     }
 
+    GitHubWebHook getHook() {
```

Comment:
:-1: I am most likely going to refactor the webhook code anyway, very risky to use mockito for this